internal/service/privatelinkendpointservice/resource_privatelink_endpoint_service_test.go

```diff
@@ -147,3 +174,20 @@ func configCompleteAWS(awsAccessKey, awsSecretKey, projectID, providerName, regi
 		}
 	`, awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID, resourceSuffix)
 }
+
+func configFailAWS(projectID, providerName, region, resourceSuffix string) string {
+	return fmt.Sprintf(`
+		resource "mongodbatlas_privatelink_endpoint" "test" {
+			project_id    = "%[1]s"
```

Comment:
we prefer %[1]q over "%[1]s", same for others

Reply:
Noted, changed `"%[1]s"` to `%[1]q` 👍